tasks/4-deploy.yaml

```
@@ -111,6 +111,12 @@ spec:
           export CHART_ROOT=$(find . -name chart)
           echo "CHART_ROOT: $CHART_ROOT"
 
+          FILE_NUM=$(find . -name Chart*.yaml |wc -l)
```

```suggestion
          FILE_NUM=$(find . -name 'Chart*.yaml' |wc -l)
```
You need to single quote it
```
root@3dd4b9127e9e:/# find . -name 'Chart*.yaml'
./Chart.yaml
./Chart344.yaml
./foo/bar/baz/Chart9395.yaml
root@3dd4b9127e9e:/# find . -name Chart*.yaml
find: paths must precede expression: `Chart344.yaml'
find: possible unquoted pattern after predicate `-name'?
root@3dd4b9127e9e:/# ls Chart*
Chart.yaml  Chart344.yaml
```

---

Done. Thanks.